src/main/java/org/terasology/launcher/game/GameDownloader.java

```diff
@@ -103,21 +104,27 @@ public boolean extractAfterDownload() {
 
     public void deleteSilentAfterExtract() {
         if (!saveDownloadedFiles) {
-            final boolean deleted = downloadZipFile.delete();
-            if (!deleted) {
-                logger.error("Could not delete downloaded ZIP file '{}'!", downloadZipFile);
+            try {
+                if (!Files.deleteIfExists(downloadZipFile)) {
```

Comment:
For me both versions are fine so I will do the revert.
I already thought about extracting a method here but wanted to check before if it even could go to the FileUtils class if something similar is done somewhere else